wp-content/themes/wisconsinwatch/functions.php

```diff
@@ -38,7 +38,7 @@ function child_stylesheet() {
 		'wcij',
 		get_stylesheet_directory_uri() . '/css/child' . $suffix . '.css',
 		null,
-		'2018-02-07'
+		'2018-10-04'
```

Comment:
We should take this opportunity to implement https://github.com/INN/umbrella-mtfp/commit/263dde3aca98bdcc75617668b4026294a025184f#diff-5e7e7a642521058e8d29dd57b65106e4R17 here, so that we don't have to keep bumping the cachebuster by hand.